WcaOnRails/app/models/competition.rb

```diff
@@ -946,15 +935,18 @@ def uses_cumulative_across_rounds?
   end
 
   # The name `is_probably_over` is meant to be surprising.
-  # We don't actually know when competitions are over, because we don't know their schedules, nor
-  # do we know their timezones.
-  # See discussion here: https://github.com/thewca/worldcubeassociation.org/pull/1206/files#r98485399.
+  # While we now know competition schedules, not all competitions might have a schedule
   def is_probably_over?
```

Comment:
Would it make sense to change this method name to `over?` now? Considering that only competitions in the past do not have a schedule, I think that would be reasonable.

Reply:
Even if we know the schedule, it might make sense to still use the word "probably", because competitions can run behind schedule =)

(Regardless of what we decide to name this method, we should definitely update the comment about the method to say that.)

Reply:
Yeah, but even in the worst case it can only be a few hours off :stuck_out_tongue: